r/neotoma_tutorial.Rmd

```diff
@@ -551,12 +650,18 @@ The majority of the taxa is made up of Mastodon samples [*Mammut americanum*](ht
 
 ## Incorporation of More Data
 
-I wanted to see if I could make sense of the habitat of these species or climate during the Pliocene to map onto the plot, but unfortunatley I could not find mappable data. I wish I had [this papers](https://www.researchgate.net/publication/37456896_Modelling_Middle_Pliocene_warm_climates_of_the_USA) data. This is a question I had many times during these this analysis.  Does anyone know where we can get predicted ancient global temperature? 
+I wanted to see if I could make sense of the habitat of these species or climate during the Pliocene to map onto the plot, but unfortunatley I could not find mappable data. I wish I had [this paper's](https://www.researchgate.net/publication/37456896_Modelling_Middle_Pliocene_warm_climates_of_the_USA) data. This is a question I had many times during these this analysis.  Does anyone know where we can get predicted ancient global temperature? 
 
 As we explore more databases we hope to gain a better understanding of the data available and how they can be combined to provide insights into our natural world. 
 
 Stay tuned for more exploration!
 
+## Activity
```

Comment:
Do you have a contribution guide/workflow for the website?

Reply:
I don't! I was thinking of updating the README.md for the website.  I put it on my to do list!